druntime/src/core/sys/posix/sys/stat.d

```diff
@@ -93,7 +93,14 @@ version (linux)
             off_t       st_size;
             blksize_t   st_blksize;
             blkcnt_t    st_blocks;
-            static if (_DEFAULT_SOURCE || _XOPEN_SOURCE >= 700)
+            version (CRuntime_Musl) {
+                c_long      __st_atime;
+                c_long      __st_atimensec;
+                c_long      __st_mtime;
+                c_long      __st_mtimensec;
+                c_long      __st_ctime;
+                c_long      __st_ctimensec;
+            } else static if (_DEFAULT_SOURCE || _XOPEN_SOURCE >= 700)
```

Comment:
This is very suspicious wrt portability. At a glance it looks like `__time_t` might just be wrong for 32-bit archs. So fix that instead?

Reply:
These are unused dummy variables in musl libc due to the switch from 32-bit `time_t` to 32-bit `time_t`. See this musl commit: https://git.musl-libc.org/cgit/musl/commit/arch/?id=38143339646a4ccce8afe298c34467767c899f51

Reply:
I changed the names to include tim32. I am uncertain if comments are worth it cause checking upstream headers seems to be an implicit assumption in the whole file.